### ISSUE TITLE
Side by side diffs layout

### ISSUE THREAD

**Issue by kegsay** [Kegsay/github-pull-review#7]
Two tables was a mistake. One table spanning lots of rows is better. Two tables was a mistake because:
- We want to word wrap lines because Ugh Scrolling. If the right side goes onto a new line but the left side doesn't... we need some communication mechanism (or mediator) between the two tables. With a single table, this Just Works.
- When you select a line to comment on, there's a ton of logic to do the same on the other table and hide the resulting duplicate comment box. Apart from being a terrible thing to do, it breaks when you preview the comment which resizes the view (needing some communication mechanism again...)
- Some code highlighters can subtly adjust the size of the text. Depending on the code on each side, this can add up to a row or two by the time you get to the bottom of the file :(

In short, **use a single table for side-by-side diffs**.

**Comment by kegsay** [comment]
This is now done.
